tests/lib/fde-setup-hook/fde-setup.go

```diff
@@ -63,17 +63,32 @@ type fdeSetupJSONStrictBase64 struct {
 }
 
 func runFdeSetup() error {
-	output, err := exec.Command("snapctl", "fde-setup-request").CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("cannot run snapctl fde-setup-request: %v", osutil.OutputErr(output, err))
+	_, initrdReleaseErr := os.Stat("/etc/initrd-release")
```

Comment:
(niptick) we already import `osutil` here so this could be: `fromInitrd := osutil.FileExists("/etc/initrd-release")`